templates/base_index.html

```diff
@@ -49,7 +49,7 @@ <h2 id="test-takeover-subtitle">Understanding real-time systems, their use cases
                     width="2382",
                     height="1684",
                     hi_def=True,
-                    loading="auto",
+                    loading="lazy",
```

Comment:
```suggestion
                    loading="auto",
```
We should keep this as auto, as it will be visible on initial page load